sql/privilege/privilege.go

```diff
@@ -42,6 +42,13 @@ const (
 	UPDATE
 )
 
+// Predefined sets of privileges.
```

Comment:
I think it's pretty clear what List does from the name. I mostly wanted the sets of privileges to be right next to the actual privileges themselves.